pkg/daemon/daemon.go

```diff
@@ -905,6 +906,12 @@ func tryEnableTun() {
 	}
 }
 
+func tryEnableVhostNet() {
+	if err := utils.LoadKernelModule("vhost-net"); err != nil {
+		glog.Errorf("tryEnableTun(): VHOST-NET kernel module not loaded: %v", err)
```

Comment:
typo: please change to `tryEnableVhostNet()`

Reply:
nice catch!